src/components/SiteHeader.tsx

```diff
@@ -6,7 +6,7 @@ import { KominfodLogo } from './KominfodLogo'
 export function SiteHeader() {
   return (
     <header className='flex h-[60px] w-full items-center bg-brand-bg px-4'>
-      <div className='mx-auto flex w-full max-w-screen-sm items-center justify-between'>
+      <div className='mx-auto flex w-full items-center justify-between'>
```

Comment:
```suggestion
      <div className='mx-auto flex w-full max-w-screen-sm items-center justify-between'>
```

This is still too big on desktop. Please at least provide a max-width.

![image](https://user-images.githubusercontent.com/5663877/182293743-2568845b-7ebc-4223-9447-0f500752d811.png)